test/mbr_update_test.rb

```diff
@@ -28,6 +31,16 @@
       )
     end
 
+    context "on s390" do
+      before do
+        allow(Yast::Arch).to receive(:architecture).and_return("s390_64")
+      end
+
+      it "does nothing except returning true" do
+        expect(subject.run).to eq true
```

Comment:
well, it is hard to test that nothing happen, but in general without mocks it will fail if it try to do anything, especially without root permissions.